daffodil-core/src/main/scala/org/apache/daffodil/grammar/GrammarTerm.scala

```diff
@@ -43,8 +44,12 @@ abstract class Gram(contextArg: SchemaComponent)
   extends OOLAGHostImpl(contextArg) {
 
   final def SDE(str: String, args: Any*): Nothing = context.SDE(str, args: _*)
+
+  @deprecated("2018-02-27", "A WarnID is now required.")
```

Comment:
Do anymore usages of this exist? Can just be removed now?

Reply:
Yes, these can be removed entirely.